spyderlib/spyder.py

```diff
@@ -693,14 +691,14 @@ def create_edit_action(text, tr_text, icon_name):
             self.debug_print("  ..tools")
             # Tools + External Tools
             prefs_action = create_action(self, _("Pre&ferences"),
-                                         icon='configure.png',
+                                         icon=ima.icon('configure'),
                                          triggered=self.edit_preferences)
             self.register_shortcut(prefs_action, "_", "Preferences")
             add_shortcut_to_tooltip(prefs_action, context="_",
                                     name="Preferences")
             spyder_path_action = create_action(self,
                                     _("PYTHONPATH manager"),
-                                    None, 'pythonpath_mgr.png',
+                                    None, icon=ima.icon('python'),

```

Comment:
This changes the `Spyder 2` theme. I mean, the Python path manager icon is now the same as the Python icon in `Spyder 2`.

I think we should leave it **exactly** as it was :-)